katalog/mimir/MAINTENANCE.md

```diff
@@ -17,6 +17,19 @@ helm template mimir-distributed /tmp/mimir-distributed -n monitoring --values MA
 
 With the `mimir-distributed-built.yaml` file, check differences with the current `deploy.yml` file and change accordingly.
 
+Move the dashboards in place:
+```bash
+cp /tmp/mimir-distributed/mixins/dashboards/*.json dashboards
+```
+
+Move the prometheus rules in place:
+```bash
+helm template mimir-distributed /tmp/mimir-distributed \
+  --set metaMonitoring.prometheusRule.enabled=true --set metaMonitoring.prometheusRule.mimirAlerts=true --set metaMonitoring.prometheusRule.mimirRules=true \
+  -s templates/metamonitoring/mixin-alerts.yaml -s templates/metamonitoring/prometheusrule.yaml -s templates/metamonitoring/recording-rules.yaml \
```

Comment:
why do we enable monitoring componets via a flag and disable them in the values file? what am I missing?

Reply:
I think it's a way to get pieces of the helm templating with -s, maybe the --set overrides stuff in the values.yaml